sphinxcontrib/confluencebuilder/config/checks.py

```diff
@@ -759,6 +759,13 @@ def conf_translate(value):
 
     # ##################################################################
 
+    # confluence_disable_appending_rest_api_to_url
+    validator.conf('confluence_disable_appending_rest_api_to_url') \
```

Comment:
(optional) Configuration should be lexicographically sorted against against all configuration entries here.